ethcore/engines/authority-round/src/lib.rs

```diff
@@ -482,6 +484,8 @@ pub struct AuthorityRound {
 	two_thirds_majority_transition: BlockNumber,
 	maximum_empty_steps: usize,
 	machine: Machine,
+	benign_reporting: Mutex<ValidatorReporting>,
+	malicious_reporting: Mutex<ValidatorReporting>,
```

Comment:
those values are set only once per lifetime of an engine - after it's creation, so they should not wrapped in the `Mutex` and should be set in `AuthorityRound::new` method